CONTRIBUTING.md

```diff
@@ -105,10 +105,11 @@ Commit types include:
 * `style`: Code style change (indentation, semicolons, ...)
 * `perf`: Performance enhancements
 * `refactor`: Code refactoring without changes to public API
+* `build`: Changes that affect the build system or external dependencies
 
 Use imperative, present tense in your commit description ("change", not "changed" or "changes") without uppercases or period (.) at the end.
 
-**Breaking changes** should be explicitly marked (with uppercase) in the message footer, e.g. `BREAKING CHANGE: <note>`. This will result in a new `MAJOR` package version.
+**Breaking changes** should be explicitly marked (singular, with uppercase) in the message footer, e.g. `BREAKING CHANGE: <note>`. This will result in a new `MAJOR` package version.
```

Comment:
perhaps make the title singular too

Reply:
It says exactly how you should write it in the example so assuming everyone actually reads the text I think it's sufficiently described. The title does not serve as an example, otherwise I would also have to uppercase it.